airline-data/src/main/scala/com/patson/data/CycleSource.scala

```diff
@@ -37,7 +37,13 @@ object CycleSource {
       insertStatement.setInt(1, cycle)
       insertStatement.executeUpdate()
       insertStatement.close()
-      
+
```

Comment:
Let's keep the setCycle w/o side effects. 

Currently we purge stuff on the cycle simulation end : https://github.com/patsonluk/airline/blob/e3f1ab40d7038512fabd73fe73920820fe8343e7/airline-data/src/main/scala/com/patson/MainSimulation.scala#L66

But i think it's probably better to do it at the `AirlineSimulation`, right before we save the balance audit perhaps